support-frontend/app/controllers/DiagnosticsController.scala

```diff
@@ -21,24 +21,30 @@ class DiagnosticsController(
     "gu.contributions.recurring.contrib-timestamp.Annual",
     "gu.contributions.contrib-timestamp",
     "gu_article_count_opt_out",
+    "GO_SO",
+  )
+  val privateCookies = List(
+    "GU_U",
+    "SC_GU_U",
   )
 
   import actionRefiners._
   def cookies(): Action[AnyContent] = NoCacheAction() { request =>
-    val cookies = request.cookies.filter { cookie =>
-      relevantCookies.contains(cookie.name)
-    }
     val humanReadableCookies =
       "Please make sure you are logged in and have visited https://www.theguardian.com and seen the issue," +
         " immediately before sending the information from this page" ::
-        s"you have ${cookies.size} cookie(s)." ::
```

Comment:
probably don't need to know the number of cookies.  Maybe that was just so we knew the reader copied and pasted the entire response correctly?